server/urls.js

```diff
@@ -40,6 +41,31 @@ function isImage(url){
 
 }
 
+function isVideo(url){
+
+	return new Promise(function(resolve, reject){
+		request(url, {timeout: RESPONSE_TIMEOUT})
+			.on('response', function(res){
+				res.on('end', () => reject('No data in response'));
+				res.destroy();
+			})
+			.on('data', function(chunk) {
```

Comment:
The `data` event fires first, then the `response` message fires afterwards. In `response` we use the `destroy` method on the `response` object to close the connection. It's the same set-up as the `isImage` function.